src/test/ui/did_you_mean/issue-54109-and_instead_of_ampersands.stderr

```diff
@@ -1,58 +1,75 @@
-error: expected `{`, found `and`
-  --> $DIR/issue-54109-and_instead_of_ampersands.rs:4:10
+error: `and` is not a logical operator
+  --> $DIR/issue-54109-and_instead_of_ampersands.rs:7:15
+   |
+LL |     let _ = a and b;
+   |               ^^^ help: instead of `and`, use `&&` to perform logical conjunction: `&&`
```

Comment:
use `span_suggestion_short` so that it hides the suggested code in the output:

```
LL |     let _ = a and b;
   |               ^^^ help: instead of `and`, use `&&` to perform logical conjunction
```

It's slightly less redundant and the tools still get the auto-applicable suggestion.